miqcli/api.py

```diff
@@ -72,21 +72,17 @@ def client(self, value):
 
     def connect(self):
         """Create a connection to the ManageIQ server.
-
-        :param settings: ManageIQ settings.
```

Comment:
Not sure why PEP8 or our checks allowed this to pass - documenting parameters that don't exist. I will find a way to block this when a PR is submitted.